### PR TITLE
Updated Forge images and resize documentation; added Windows and other 2.0 references

### PR DESCRIPTION
**Pull request by ryanbooz**
Added updated images to Forge documentation based on the new UI and a section that discusses resizing.

Added link back to Install page to show Windows and added text on each install page to reference updating to 2.0 via the separate document.

### CODE REVIEW

File: getting-started/exploring-forge.md [hunk 1]
```diff
@@ -94,6 +93,56 @@ continue to test your use case. Before the end of your trial, we encourage you
...
+Before you modify the compute or storage settings for a Forge Service, please note the following limitations
+and when a change to these settings will result in momentary downtime.
+
+**Storage**: Storage changes are applied with no downtime, typically available within a few seconds. Other
+things to note about storage changes:
+ * At the current time, storage can only be increased in size.
+ * Storage size changes can only be made once every six (6) hours.
+ * Storage can be modified in 25GB increments between 25GB-500GB, and then in 100GB increments between 500GB-1TB.
+
+**Compute**: Modifications to the compute settings of your service (up or down) can be applied at any time, however,
```
Comment (mfreed) [inline getting-started/exploring-forge.md:111]:
```suggestion
**Compute**: Modifications to the compute size of your service (increases or decreases) can be applied at any time, however,
```

File: getting-started/exploring-forge.md [hunk 2]
```diff
@@ -29,16 +29,15 @@ and, failing that, please [contact us][contact-timescale].
...
 1. Click 'Create service' once your configuration is complete.
 
-We'll be adding additional CPU and storage configurations in the future.  We'll
-also be adding the ability to quickly switch CPU configurations and storage
-options for your existing database with just a click, so you can scale with
-your usage needs.  For now, please [reach out][contact-timescale] if you need
-to reconfigure an existing service.
+>:TIP:Don't worry if too much about the size settings that you choose initially. With Timescale Forge,
+it's easy to modify both the compute (CPU/Memory) and storage associated with the service
+that you just created. As you get to know Timescale and how your data processing needs vary,
```
Comment (mfreed) [inline getting-started/exploring-forge.md:39]:
```suggestion
that you just created. As you get to know TimescaleDB and how your data processing needs vary,
```

File: getting-started/exploring-forge.md [hunk 3]
```diff
@@ -94,6 +93,56 @@ continue to test your use case. Before the end of your trial, we encourage you
...
+**Storage**: Storage changes are applied with no downtime, typically available within a few seconds. Other
+things to note about storage changes:
+ * At the current time, storage can only be increased in size.
+ * Storage size changes can only be made once every six (6) hours.
+ * Storage can be modified in 25GB increments between 25GB-500GB, and then in 100GB increments between 500GB-1TB.
+
+**Compute**: Modifications to the compute settings of your service (up or down) can be applied at any time, however,
+please note the following:
+ * **_There will be momentary downtime_** while the compute settings are applied. In most cases, this downtime
+will be less than a minute.
```
Comment (mfreed) [inline getting-started/exploring-forge.md:114]:
```suggestion
will be less than 30 seconds.
```

File: getting-started/installation-apt-debian.md [hunk 4]
```diff
@@ -42,6 +42,30 @@ sudo apt-get update
...
+The following NEW packages will be installed:
+  timescaledb-2-loader-postgresql-12 timescaledb-2-postgresql-12
+0 upgraded, 2 newly installed, 2 to remove and 11 not upgraded.
+Need to get 953 kB of archives.
+After this operation, 1314 kB of additional disk space will be used.
+Do you want to continue? [Y/n]
+```
+
+Once you confirm and install the newest binary package, you must still perform the
+EXTENSION update as discussed in [Updating Timescale to 2.0][update-tsdb-2]
```
Comment (mfreed) [inline getting-started/installation-apt-debian.md:67]:
```suggestion
EXTENSION update as discussed in [Updating Timescale to 2.0][update-tsdb-2].
```

File: getting-started/installation-apt-debian.md [hunk 5]
```diff
@@ -66,4 +90,5 @@ sudo service postgresql restart
 [config]: /getting-started/configuring
 [contact]: https://www.timescale.com/contact
 [slack]: https://slack.timescale.com/
-[multi-node-basic]: /getting-started/setup-multi-node-basic
\ No newline at end of file
+[multi-node-basic]: /getting-started/setup-multi-node-basic
+[update-tsdb-2]: /update-timescaeldb/update-tsdb-2
```
Comment (mfreed) [inline getting-started/installation-apt-debian.md:94]:
```suggestion
[update-tsdb-2]: /update-timescaledb/update-tsdb-2
```

File: getting-started/installation-apt-ubuntu.md [hunk 6]
```diff
@@ -43,6 +43,30 @@ sudo apt-get update
...
+The following NEW packages will be installed:
+  timescaledb-2-loader-postgresql-12 timescaledb-2-postgresql-12
+0 upgraded, 2 newly installed, 2 to remove and 11 not upgraded.
+Need to get 953 kB of archives.
+After this operation, 1314 kB of additional disk space will be used.
+Do you want to continue? [Y/n]
+```
+
+Once you confirm and install the newest binary package, you must still perform the
+EXTENSION update as discussed in [Updating Timescale to 2.0][update-tsdb-2]
```
Comment (mfreed) [inline getting-started/installation-apt-ubuntu.md:68]:
```suggestion
EXTENSION update as discussed in [Updating Timescale to 2.0][update-tsdb-2].
```

File: getting-started/installation-apt-ubuntu.md [hunk 7]
```diff
@@ -68,4 +92,5 @@ sudo service postgresql restart
 [config]: /getting-started/configuring
 [contact]: https://www.timescale.com/contact
 [slack]: https://slack.timescale.com/
-[multi-node-basic]: /getting-started/setup-multi-node-basic
\ No newline at end of file
+[multi-node-basic]: /getting-started/setup-multi-node-basic
+[update-tsdb-2]: /update-timescaeldb/update-tsdb-2
```
Comment (mfreed) [inline getting-started/installation-apt-ubuntu.md:96]:
```suggestion
[update-tsdb-2]: /update-timescaledb/update-tsdb-2
```

File: getting-started/installation-source-windows.md [hunk 8]
```diff
@@ -64,6 +64,11 @@ shared_preload_libraries = 'timescaledb'
 
 Then, restart the PostgreSQL instance.
 
+#### Updating from TimescaleDB 1.x to 2.0
+Once the latest TimescaleDB 2.0 are installed, you can update the EXTENSION 
+in your database as discussed in [Updating Timescale to 2.0][update-tsdb-2]
```
Comment (mfreed) [inline getting-started/installation-source-windows.md:69]:
```suggestion
in your database as discussed in [Updating Timescale to 2.0][update-tsdb-2].
```

File: getting-started/installation-source-windows.md [hunk 9]
```diff
@@ -73,3 +78,4 @@ to `bootstrap`.
 [CMake]: https://cmake.org/
 [github-releases]: https://github.com/timescale/timescaledb/releases
 [github-timescale]: https://github.com/timescale/timescaledb
+[update-tsdb-2]: /update-timescaeldb/update-tsdb-2
```
Comment (mfreed) [inline getting-started/installation-source-windows.md:81]:
```suggestion
[update-tsdb-2]: /update-timescaledb/update-tsdb-2
```

File: getting-started/installation-windows.md [hunk 10]
```diff
@@ -28,6 +23,10 @@ Press ENTER/Return key to close...
 ```
 Go ahead and press ENTER to close the window
 
+#### Updating from TimescaleDB 1.x to 2.0
+Once the latest TimescaleDB 2.0 are installed, you can update the EXTENSION 
+in your database as discussed in [Updating Timescale to 2.0][update-tsdb-2]
```
Comment (mfreed) [inline getting-started/installation-windows.md:28]:
```suggestion
in your database as discussed in [Updating Timescale to 2.0][update-tsdb-2].
```

File: getting-started/installation-windows.md [hunk 11]
```diff
@@ -54,3 +53,4 @@ to `bootstrap`.
 [config]: /getting-started/configuring
 [contact]: https://www.timescale.com/contact
 [slack]: https://slack.timescale.com/
+[update-tsdb-2]: /update-timescaeldb/update-tsdb-2
```
Comment (mfreed) [inline getting-started/installation-windows.md:56]:
```suggestion
[update-tsdb-2]: /update-timescaledb/update-tsdb-2
```

File: getting-started/installation-yum.md [hunk 12]
```diff
@@ -55,6 +55,10 @@ sudo yum update -y
 sudo yum install -y timescaledb-2-postgresql-:pg_version:
 ```
 
+#### Upgrading from TimescaleDB 1.x
+If you are upgrading from TimescaleDB 1.x, the EXTENSION must be updated 
+in the database as discussed in [Updating Timescale to 2.0][update-tsdb-2]
```
Comment (mfreed) [inline getting-started/installation-yum.md:60]:
```suggestion
in the database as discussed in [Updating Timescale to 2.0][update-tsdb-2].
```

File: getting-started/installation-yum.md [hunk 13]
```diff
@@ -80,4 +84,5 @@ refer to your distribution for how to restart services.
 [createuser]: https://www.postgresql.org/docs/current/sql-createrole.html
 [contact]: https://www.timescale.com/contact
 [slack]: https://slack.timescale.com/
-[multi-node-basic]: /getting-started/setup-multi-node-basic
\ No newline at end of file
+[multi-node-basic]: /getting-started/setup-multi-node-basic
+[update-tsdb-2]: /update-timescaeldb/update-tsdb-2
```
Comment (mfreed) [inline getting-started/installation-yum.md:88]:
```suggestion
[update-tsdb-2]: /update-timescaledb/update-tsdb-2
```

**Review by mfreed — APPROVED** [review]

**Review by RobAtticus — APPROVED** [review]

File: getting-started/installation-apt-debian.md [hunk 14]
```diff
@@ -42,6 +42,30 @@ sudo apt-get update
 sudo apt-get install timescaledb-2-postgresql-:pg_version:
 ```
 
+#### Upgrading from TimescaleDB 1.x
+If you are upgrading from TimescaleDB 1.x, the `apt` package will as to first
```
Comment (solugebefola) [inline getting-started/installation-apt-debian.md:46]: > the `apt` package _will as to first_

Not sure what this phrase was meant to be?

File: getting-started/installation-apt-debian.md [hunk 15]
```diff
@@ -42,6 +42,30 @@ sudo apt-get update
...
+  timescaledb-loader-postgresql-12 timescaledb-postgresql-12
+The following NEW packages will be installed:
+  timescaledb-2-loader-postgresql-12 timescaledb-2-postgresql-12
+0 upgraded, 2 newly installed, 2 to remove and 11 not upgraded.
+Need to get 953 kB of archives.
+After this operation, 1314 kB of additional disk space will be used.
+Do you want to continue? [Y/n]
+```
+
+Once you confirm and install the newest binary package, you must still perform the
```
Comment (solugebefola) [inline getting-started/installation-apt-debian.md:66]:
```suggestion
Once you confirm and install the newest binary package, perform the
```

**Pull request merged**